--- libarchive-3.3.2/libarchive/archive_read_support_filter_program.c~	2017-02-05 19:28:19.000000000 +0000
+++ libarchive-3.3.2/libarchive/archive_read_support_filter_program.c	2018-06-04 09:38:35.233221361 +0000
@@ -397,7 +397,7 @@
 __archive_read_program(struct archive_read_filter *self, const char *cmd)
 {
 	struct program_filter	*state;
-	static const size_t out_buf_len = 65536;
+	static const size_t out_buf_len = 65536 * 8;
 	char *out_buf;
 	const char *prefix = "Program: ";
 	pid_t child;
